schema_based_on_diagram.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE patients(
+  id int PRIMARY KEY,
+    name VARCHAR(255),
+    date_of_birth DATE,
+)
+
+-------------------------------
+CREATE TABLE medical_histories(
+    id int PRIMARY KEY,
+    admitted_at timestamp,
+    patients_id int,
+    status VARCHAR
+)
+
+----------------------------
+
+CREATE TABLE invoices(
+    id int PRIMARY KEY,
+    total_amount decimal,
+    generated_at timestamp,
+    payed_at timestamp,
+    medical_history_id int,
+    constraint fk_invoices_medical_history foreign key(medical_history_id) references medical_histories(id)
+)
+
+create table treatment (
```

Comment:
Remember that we need to follow the naming conventions as it is on the `ERD` since this is really important when writing different naming as can be singular or plural can give us an error in big applications

- Kindly fix the name of the table to be `treatments` to match the `ERD` provided 🥇